bash-preexec.sh

```diff
@@ -239,6 +245,7 @@ __bp_preexec_invoke_exec() {
     local this_command
     this_command=$(
         export LC_ALL=C
+		# shellcheck disable=SC1007
         HISTTIMEFORMAT= builtin history 1 | sed '1 s/^ *[0-9][0-9]*[* ] //'
```

Comment:
Does this work?

```suggestion
        HISTTIMEFORMAT='' builtin history 1 | sed '1 s/^ *[0-9][0-9]*[* ] //'
```

Reply:
It passes the tests and is syntactically the same, I see no reason why it shouldn't work.
In fact the reason for the change is that this is clearer.

Explicitly setting HISTTIMEFORMAT to the empty string '' seems clearer than the old text to me.